comp/dogstatsd/server/server_test.go

```diff
@@ -1038,3 +1028,8 @@ func requireStart(t *testing.T, s Component, demux aggregator.Demultiplexer) {
 	assert.NotNil(t, s)
 	assert.True(t, s.IsRunning())
 }
+
+func createDemultiplexer(t *testing.T) demultiplexer.FakeSamplerMock {
+	return fxutil.Test[demultiplexer.FakeSamplerMock](t, logimpl.MockModule(),
+		demultiplexerimpl.FakeSamplerMockModule())
+}
```

Comment:
In this file the demux used to have a flush interval of 10ms but I believe the mock is always initialized with 1 hour (which was the default value and used in most files)

Reply:
I did that on purpose. I have tried to limit the difference between tests. Locally, tests passed. I will fix them if `1 hour` was really needed`